Bugs2

Hildthelsta opened the pull request:
Part of Fixes

Hildthelsta committed:
Correction of Bug .02 and .04  (#71)

* Solved Bug.04

- Solved Bug.04 #66

* Removed Bug

- Removed bug where student label became white when Admin was selected in the login page.

* Solved Bug.02 #58

-Added alert when the organizer reaches 2000 char in the description during the creation of a new event
-Hard coded description 2000 char limit

* Fixed bug.02 but for Location and Title

---------

Co-authored-by: Hildthelsta <d_cazac@live.concordia.ca>

Hildthelsta committed:
StudentID Validation

- Validate if the ID that the student enter isn't already in use

Hildthelsta committed:
-Deal Overflow

-Overflow in Student
-Overflow in Signed Out
-Overflow in Organizer
-Overflow in Admin

Hildthelsta committed:
AddDemoUsers

Hildthelsta committed:
Fixes

- Check rejections status during login
- Solved disappearing problem when downloading csv in admin
- Fixed ticket count disappearing when going to an empty pastcount

Hildthelsta committed:
Merge branch 'main' into bugs2

Hildthelsta committed:
Resolve problem from check

Hildthelsta committed:
Fixed crash from drivers

Hildthelsta committed:
Merge branch 'main' into bugs2

Hildthelsta committed:
Re-solved the program of role verification.

Hildthelsta committed:
-Remove db for better merge

Hildthelsta committed:
- Dates for sorting events (Student&Logout)

Hildthelsta committed:
-Hackathon as 'Other' instead of 'Competition'

Hildthelsta committed:
Error Message

-Error message for none valid phone number or email for student and organizer

Hildthelsta committed:
-Remove db

Make easier to merge with main

Hildthelsta committed:
Merge branch 'main' into bugs2

Hildthelsta committed:
Corrected incongruency found by .Net Build

Hildthelsta committed:
Fixes

-Can now move in the tabs of Organizer/Events
- Limit description of new org to 3k and title to 300
-Corrected vizual bug in Student/EventDetails caused by a z parameter

PenguKev reviewed (approved):
the search rebuged it self, but if its fixed in main then merge carefully

Hildthelsta committed:
UI Correction for Student/Events

- Fix because it's not scroll, but select for dates

Hildthelsta committed:
Fixes

-Repair Org search in Student/Events
-upper and lower case return the same event

mistersuun reviewed (approved):
good PR
